applications/NXmx.nxdl.xml

```diff
@@ -36,22 +36,29 @@
 				images, using one of the indices to select a detector module.
 			</doc>
 			<symbol name="dataRank">
-				<doc>rank of the ``data`` field</doc>
+				<doc>Rank of the ``data`` field</doc>
```

Comment:
Must be 3 or greater.

Reply:
after discussion with @yayahjb it was decided that the constraint was not required